
FAIL at (core_test.clj:34)
    Expected: 3
      Actual: 4
^^^^ The previous failure was expected ^^^^

FAIL at (core_test.clj:45)
Actual result did not agree with the checking function.
        Actual result: 4
    Checking function: odd?
^^^^ The previous failure was expected ^^^^

FAIL at (core_test.clj:56)
Actual result did not agree with the checking function.
        Actual result: a function named 'even?'
    Checking function: (exactly odd?)
^^^^ The previous failure was expected ^^^^

FAIL at (core_test.clj:74)
Actual result did not agree with the checking function.
        Actual result: [3 3 1 2]
    Checking function: (just [1 2 3] :in-any-order)
    The checker said this about the reason:
        Expected three elements. There were four.
^^^^ The previous failure was expected ^^^^

FAIL at (core_test.clj:120)
You claimed the following was needed, but it was never used:
    (alive? ...cell...)

FAIL at (core_test.clj:121)
You claimed the following was needed, but it was never used:
    (neighbor-count ...cell...)

FAIL at (core_test.clj:118)
Actual result did not agree with the checking function.
        Actual result: nil
    Checking function: truthy
^^^^ The previous three failures were expected ^^^^
-----
-- WARNING: In this version, prerequisite arguments that are functions are
-- considered checkers. In the near future, they'll be considered ordinary
-- values. If you want them to be checkers, you need to declare them as such.
-- The function argument I'm objecting to is:
--     a function named 'odd?'
--     near (core_test.clj:197)
-- To future-proof your fact (and make this message go away), do this:
--     (as-checker <<<your function argument>>>)
-- If you never wanted this argument to be treated as a checker, do this:
       (exactly <<<your function argument>>>)
-- For more, see https://github.com/marick/Midje/wiki/Checkers-within-prerequisites
-----

FAIL at (core_test.clj:227)
You claimed the following was needed, but it was never used:
    (property-of ...data-structure-value-1...)

FAIL at (core_test.clj:227)
You claimed the following was needed, but it was never used:
    (data-structure)
^^^^ The previous two failures were expected ^^^^

Testing basic.core-test

Ran 0 tests containing 0 assertions.
0 failures, 0 errors.
{:type :summary, :test 0, :pass 0, :fail 0, :error 0}
